# HG changeset patch
# User maldun <domors@gmx.net>
# Date 1283560011 -7200
# Node ID 257df6b32dae1ed68f2a0d68f2e93332eef2393f
# Parent  5b338f2e484f2065d3d30d47bc204d6e9ed13d12
Trac 9808: Updatet 2 outdated doctests in doc/en/faq/faq-usage.rst
and sage/matrix/matrix1.pyx

diff -r 5b338f2e484f -r 257df6b32dae doc/en/faq/faq-usage.rst
--- a/doc/en/faq/faq-usage.rst	Thu Aug 05 03:35:44 2010 -0700
+++ b/doc/en/faq/faq-usage.rst	Sat Sep 04 02:26:51 2010 +0200
@@ -339,7 +339,6 @@
     sage: RealNumber = float; Integer = int
     sage: from scipy import stats
     sage: stats.ttest_ind(list([1,2,3,4,5]),list([2,3,4,5,.6]))
-    doctest...DeprecationWarning...
     (0.076752955645333687, 0.94070490247380478)
     sage: stats.uniform(0,15).ppf([0.5,0.7])
     array([  7.5,  10.5])
diff -r 5b338f2e484f -r 257df6b32dae sage/matrix/matrix1.pyx
--- a/sage/matrix/matrix1.pyx	Thu Aug 05 03:35:44 2010 -0700
+++ b/sage/matrix/matrix1.pyx	Sat Sep 04 02:26:51 2010 +0200
@@ -446,7 +446,7 @@
 
             sage: import numpy
             sage: sorted(numpy.typecodes.items())
-            [('All', '?bhilqpBHILQPfdgFDGSUVO'), ('AllFloat', 'fdgFDG'), ('AllInteger', 'bBhHiIlLqQpP'), ('Character', 'c'), ('Complex', 'FDG'), ('Float', 'fdg'), ('Integer', 'bhilqp'), ('UnsignedInteger', 'BHILQP')]
+            [('All', '?bhilqpBHILQPfdgFDGSUVOMm'), ('AllFloat', 'fdgFDG'), ('AllInteger', 'bBhHiIlLqQpP'), ('Character', 'c'), ('Complex', 'FDG'), ('Datetime', 'Mm'), ('Float', 'fdg'), ('Integer', 'bhilqp'), ('UnsignedInteger', 'BHILQP')]
 
         Alternatively, numpy automatically calls this function (via
         the magic :meth:`__array__` method) to convert Sage matrices
